examples/AdminClient/Program.cs

```diff
@@ -30,7 +30,7 @@ public class Program
 
         static void ListGroups(string brokerList)
         {
-            using (var adminClient = new AdminClient(new AdminClientConfig { BootstrapServers = brokerList }))
+            using (var adminClient = new AdminClientBuilder(new AdminClientConfig { BootstrapServers = brokerList }).Build())
```

Comment:
Now with the builder pattern in place, isn't there a more direct way to do this? E.g.,
`new AdminClientBuilder().SetBootstrapServers(brokerList).Build()`
or perhaps
`new AdminClientBuilder().SetConfig("bootstrap.servers", brokerList).Build()`

Reply:
no. but yeah - you're right, we don't want the strongly typed config classes, rather we'd be better off with strongly typed setter methods on the builder. we probably don't want to maintain both (one or the other). note that the current implementation can grow into this (the config is specified as an `IEnumerable<KeyValuePair<string,string>>`, which we will want to leave), but we'd have the kind of superfluous config classes.

This would actually be a fair bit of work to change - the config generator code is pretty fiddly. Let's discuss what we want to do here later.

Reply:
I do like the strongly typed config classes, and I think they are useful for more complex program setups.
What I reacted to here was that the simplicity of 'just creating a client' is somewhat lost here with the verbosity of `new AdminClientConfig { ... }`.

Not sure I have a better idea though.

Reply:
yeah it is annoying. and yeah, i'm talking with too much conviction above. it would be a bit annoying to have two ways of doing the same thing, but I guess there's a reason I strongly preferred the simple property setters for such a long time. We could just add a `SetConfig` method, but that wouldn't be strongly typed, which would be a shame. So maybe a full array of Set methods is on the cards for the future. Or maybe we just add a subset (bootstrap servers, group id). That might actually be optimal actually.